git-pr-merge

```diff
@@ -108,7 +113,6 @@ prmerge() {
   common::setup ${args[@]+"${args[@]}"}
   prmerge::prepare
   prmerge::merge
-  prmerge::push
```

Comment:
Sorry, i don't get write why this is removed.

Reply:
The functionality of `prmerge::push` got moved into `local_merge`, as it is really part of the merge process. Merge already has different paths for squash merging vs local merging + push, so it seemed cleaner that way.

For this change, I had to test whether you could actually merge a PR from a fork this way - i.e. will it actually merge and close the PR if you push the merge commit from another repo. It does work but that is when I realised splitting the push from the merge seemed wrong.